src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2765,6 +2772,16 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-lookup"
+    title={<InlineCode>lookup()</InlineCode>}
```

Comment:
`lookup()` doesn't really belong under "Non-aggregatore functions" - that implies it can be used in a select, where, or facet clause, which is not possible.  Since that is just a link to another page, should it go inside the FROM section? `lookup()` can _only_ be used in the `FROM` clause, and it is the first, one, and _only_ function that can be used in the FROM clause.

Also, most function docs show the args of the functions int he title.  Should we make the title of this (wherever it ends up) `lookup(table)`?